### DISCUSSION TITLE
Draft proposal review

### DISCUSSION THREAD

**Discussion by srini047** [jina-ai/GSoC#28] [Ideas]
Hey team!! This is [Sriniketh J](https://github.com/srini047)🇮🇳, a Junior from PSG iTech pursuing Electronics and Communication Engineering. I am an OS enthusiast and worked previously on Jina and have been a part of this community for quite almost a year. Looking forward to working on **Jax Support in DocArray v2**, as this interests me a lot and the Jax framework looks very fresh and interesting to work with a research perspective giving it higher prospects.

**[Draft proposal](https://docs.google.com/document/d/1quWaFQuVfkq7MH_w_yJQr2Os6jvYrfhi_XO-NPu_uUw/edit?usp=sharing)**

Would like to get my proposal reviewed the earliest to make further changes. Up for any kind of appreciation, suggestions, and criticism.

Thanks in advance🙏

**Comment by hey-sa** [comment]
Hi @srini047 . Thank you for sharing the proposal for the project.

Hi hi @samsja , what do you think of the proposal, do you have any feedback or suggestions on how to make this proposal more compelling? Thanks for your time.
